src/python/pants/bin/daemon_pants_runner.py

```diff
@@ -161,7 +161,7 @@ def _tty_stdio(cls, env):
       'please file a bug at http://github.com/pantsbuild/pants'
       .format([stdin_ttyname, stdout_ttyname, stderr_ttyname])
     )
-    with open(stdin_ttyname, 'rb+wb', 0) as tty:
+    with open(stdin_ttyname, 'rb+', 0) as tty:
```

Comment:
`rb+wb` isn't allowed in Py3. I think `rb+` gets the job done. See https://stackoverflow.com/questions/6648493/open-file-for-both-reading-and-writing